From 98db4ed54bd84d9b9616a44c41d18840ed94433a Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:23:09 -0400
Subject: [alsa] rme32: Use standard printk helpers

Message-id: <140972823573-167-patchset-jkysela@redhat.com>
Patchwork-id: 90867
O-Subject: [RHEL 7.1 PATCH 166/442] ALSA: rme32: Use standard printk helpers
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Convert with dev_err() and co from snd_printk(), etc.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 03952a3e2d98757a848a0d299d3e7c8ad0f08ffc)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/rme32.c b/sound/pci/rme32.c
index 77465cc..cc2f0c1 100644
--- a/sound/pci/rme32.c
+++ b/sound/pci/rme32.c
@@ -1349,14 +1349,15 @@ static int snd_rme32_create(struct rme32 *rme32)
 
  rme32->iobase = ioremap_nocache(rme32->port, RME32_IO_SIZE);
  if (!rme32->iobase) {
-  snd_printk(KERN_ERR "unable to remap memory region 0x%lx-0x%lx\n",
+  dev_err(rme32->card->dev,
+   "unable to remap memory region 0x%lx-0x%lx\n",
       rme32->port, rme32->port + RME32_IO_SIZE - 1);
   return -ENOMEM;
  }
 
  if (request_irq(pci->irq, snd_rme32_interrupt, IRQF_SHARED,
    KBUILD_MODNAME, rme32)) {
-  snd_printk(KERN_ERR "unable to grab IRQ %d\n", pci->irq);
+  dev_err(rme32->card->dev, "unable to grab IRQ %d\n", pci->irq);
   return -EBUSY;
  }
  rme32->irq = pci->irq;
-- 
1.7.1